planprogenerator/planproxml/edgexml.py

```diff
@@ -35,20 +36,20 @@ def get_top_edge_xml(edge):
                + f"                  <Wert>{connection_b}</Wert>\n" \
                + f"                </TOP_Anschluss_B>\n" \
                + f"                <TOP_Laenge>\n" \
-               + f"                  <Wert>{edge.get_length():.3f}</Wert>\n" \
+               + f"                  <Wert>{edge.length:.3f}</Wert>\n" \
                + f"                </TOP_Laenge>\n" \
                + f"              </TOP_Kante_Allg>\n" \
                + f"            </TOP_Kante>\n"
 
     @staticmethod
-    def get_geo_edge_xml(geo_node_a, geo_node_b, uuid, top_edge):
-        min_x = min(geo_node_a.x, geo_node_b.x)
-        min_y = min(geo_node_a.y, geo_node_b.y)
-        max_x = max(geo_node_a.x, geo_node_b.x)
-        max_y = max(geo_node_a.y, geo_node_b.y)
+    def get_geo_edge_xml(geo_node_a: GeoNode, geo_node_b: GeoNode, uuid, top_edge: Edge):
+        min_x = min(geo_node_a.geo_point.x, geo_node_b.geo_point.x)
+        min_y = min(geo_node_a.geo_point.y, geo_node_b.geo_point.y)
+        max_x = max(geo_node_a.geo_point.x, geo_node_b.geo_point.x)
+        max_y = max(geo_node_a.geo_point.y, geo_node_b.geo_point.y)
         length = math.sqrt(math.pow(max_x - min_x, 2) + math.pow(max_y - min_y, 2))
```

Comment:
This part should also be moved to some kind of library depending of the configuration.